DATA_OVERLAY.md

```diff
@@ -57,13 +58,19 @@ As shown by the figure below, the histogram of scaled data is more evenly distri
 
 ![Histogram of HPA scaled data](./assets/img/hpaRNA-TPM-scaleddata-histogram.png)
 
+
+### Linear distribution of label colors
+
+When a dataset contains labels a simplay way to assign values is to split the range `[0,1]` in `N` parts with a fixed step size of `1/N`, where `N` is the number of labels. Then for each entry calculate `scaled_data = indexOf(label)/N`, where `label` is the label of the entry.
```

Comment:
_Simplay_ - is this a typo or a term I do not know?

Reply:
Yes. Thank you!